examples/bastion/config.json

```diff
@@ -1,86 +1,56 @@
 {
     "location": "variables.location",
     "resource_group": "variables.resource_group",
-    "install_from": "jumpbox",
-    "admin_user": "hpcadmin",
+    "install_from": "",
+    "admin_user": "variables.admin_user",
     "variables": {
-        "hpc_image": "OpenLogic:CentOS-HPC:7.7:latest",
-        "win_image": "MicrosoftWindowsDesktop:Windows-10:rs5-pron:latest",
-        "win_password": "<NOT-SET>",
+        "jumpbox_image": "OpenLogic:CentOS:7.7:latest",
+        "admin_user": "hpcadmin",
+        "win_image": "MicrosoftWindowsDesktop:Windows-10:21h1-pron:latest",
         "location": "<NOT-SET>",
         "resource_group": "<NOT-SET>",
-        "vm_type": "<NOT-SET>",
+        "vm_type": "Standard_D4s_v3",
+        "vnet_name": "BastionVNet",
         "low_priority": false,
-        "bastion_subnet": "10.2.2.0/27",
+        "bastion_subnet": "10.10.0.0/26",
```

Comment:
I wonder should we change the address space to be 10.201.0.0/20 so I will work also work with current restricted networking security rules.